docs/source/installation.rst

```diff
@@ -5,9 +5,8 @@
 Installation
 ============
 
-You have several options you may use to install the
-**IBM z/OS Core Collection**. These are Ansible Galaxy,
-Ansible Automation Hub, and a local build.
+You can install the **IBM z/OS Core Collection** using one of these options:
+Ansible Galaxy, Ansible Automation Hub, or a local build.
 
 For more information on installing collections, see `using collections`_.
```

Comment:
i see a lot of mixed case references to IBM z/OS core collection/Core Collection. We are also referring to it as a product name and use it generically and use "the IBM z?OS core collection." We have to come to an agreement on this

Reply:
I tried to correct them all in the last release, the request came late so I might have missed some but the collection should be written and referred to as "IBM z/OS core collection"  <--- exactly with this letter case where "c"'s are lower case, with the forward slash between z/OS".

Reply:
ack! let me revisit all instances and update.